src/odb/src/db/dbSite.h

```diff
@@ -94,6 +141,7 @@ inline dbIStream& operator>>(dbIStream& stream, _dbSite& site)
   stream >> site._height;
   stream >> site._width;
   stream >> site._next_entry;
+  stream >> site._row_pattern;
```

Comment:
This is a schema change.  This needs to be conditional on the schema version which needs to be incremented.